nodewatch/Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+defaultCiPipeline {
+	platform = ['ubuntu']
+	ciBuildDockerfile = 'python.Dockerfile'
+
+	packageId = 'nodewatch-puller'
+
+	codeCoverageTool = 'coverage'
+	minimumCodeCoverage = 95
+}
```

Comment:
Should remove the code coverage settings unless you plan to add tests.

```
	codeCoverageTool = 'coverage'
	minimumCodeCoverage = 95
```

Reply:
I will comment this line